README.md

```diff
@@ -1,4 +1,6 @@
-Mock library for Dart inspired by [Mockito](https://github.com/mockito/mockito).
+# Mock library for Dart inspired by [Mockito](https://github.com/mockito/mockito)
+
+**NOTE:** This is not an official Google product
```

Comment:
I would prefer that this text remain at the bottom of the README. This is pretty standard across our open source projects.